macos/src/main/java/com/github/weisj/darklaf/platform/macos/ui/MacOSDecorationsUtil.java

```diff
@@ -61,6 +61,11 @@ protected static DecorationInformation installDecorations(final JRootPane rootPa
             boolean isDarkTheme = UIManager.getBoolean("Theme.dark");
             JNIDecorationsMacOS.setDarkTheme(windowHandle, isDarkTheme);
         }
+        if (jniInstall) {
+            Dimension size = window.getSize();
+            size.height += titleBarHeight;
+            window.setSize(size);
+        }
```

Comment:
I guess this should be removed.

Reply:
Does it work without it then?

Reply:
Yes, it works great without this `setSize`